tests/proxy.spec.ts

```diff
@@ -75,6 +76,64 @@ it('should work with IP:PORT notion', async ({browserType, browserOptions, serve
   await browser.close();
 });
 
+it('should not work with localhost by default on Chromium', async ({ browserType, browserOptions }) => {
+  let proxiedRequest = false;
+  const server = net.createServer(socket => {
+    socket.on('data', () => {
+      proxiedRequest = true;
+      socket.end();
+    });
+  });
+  await new Promise(f => server.listen(0, f));
+
+  const opts: LaunchOptions = {
+    ...browserOptions,
+    proxy: { server: `http://127.0.0.1:${(server.address() as any).port}` },
```

Comment:
You can also pass this directly in the [proxy options](https://playwright.dev/docs/api/class-browser#browser-new-context-option-proxy). Passing the arg directly is not needed.

```suggestion
    proxy: { server: `http://127.0.0.1:${(server.address() as any).port}`, bypass: '<-loopback>' },
```